### PR TITLE
[stable/moodle] - Expose liveness/readiness probes to values.yaml

### PR DESCRIPTION
**Pull request by tompizmor**
**What this PR does / why we need it**:
This PR expose the configuration of Moodle liveness and readiness probes to the values.yaml.
It also updates the MariaDB dependency as it was quite old.

### CODE REVIEW

File: stable/moodle/requirements.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 dependencies:
 - name: mariadb
-  version: 0.7.0
+  version: 3.x.x
```
Comment (sameersbn) [inline stable/moodle/requirements.yaml]: Locking to a major version series feels like a very risky move. At best we should lock down to a minor version. 
Having said that I think we no longer suggest locking to version series. I can't remember the exact reason /cc @prydonius

**Review by sameersbn — CHANGES_REQUESTED** [review]

File: stable/moodle/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: moodle
-version: 1.0.3
+version: 1.0.4
```
Comment (sameersbn) [inline stable/moodle/Chart.yaml:2]: the version bump in the mariadb dependency warrants a major version bump in the moodle chart as it makes the release backwards incompatible.

**Review by sameersbn — CHANGES_REQUESTED** [review]

File: stable/moodle/requirements.yaml (hunk 1 above)
Comment (tompizmor) [reply stable/moodle/requirements.yaml]: There shouldn't be any new MariaDB release matching 3.x.x as we are already in the 4.x.x version.
I tested the chart with this specific version of MariaDB and it works fine

File: stable/moodle/Chart.yaml (hunk 2 above)
Comment (tompizmor) [reply stable/moodle/Chart.yaml:2]: I don't understand why updating the dependency requires a major version bump in the chart. The chart continues working without any change.

Comment (sameersbn) [reply stable/moodle/Chart.yaml:2]: the mariadb chart used here has major changes (master/slave replication by default). A user of the moodle-1.0.3 chart would not simply be able to perform a upgrade to moodle-1.0.4 as the mariadb data will not migrate automatically to the new topology automatically.

Comment (tompizmor) [reply stable/moodle/Chart.yaml:2]: that changes are for mariadb 4.x, but it is true that with the 3.x version the moodle upgrade does not work well. At the moment I am undoing the mariadb updates changes and just expose the readiness/liveness probes

Comment (carrodher) [reply stable/moodle/Chart.yaml:2]: We'll update MariaDB in a different way. Reverted f703b9f

**Pull request merged**